src/simulator/interpret.cpp

```diff
@@ -7,6 +7,26 @@ using namespace std;
 
 namespace CoreIR {
 
+  BitVector BfloatRoundToNearestEven(const BitVector& val) {
+    assert(val.bitLength() == 32);
+
+    auto half = BitVector(32, 0x0000800);
```

Comment:
This bitvector should have an additional zero: 
`auto half = BitVector(32, 0x00008000);`

Reply:
Good catch, fixing